sdk/storage/azblob/constants.go

```diff
@@ -6,33 +6,49 @@
 
 package azblob
 
+var SASVersion = "2019-12-12"
+
 //nolint
 const (
-	SASVersion = "2019-12-12"
-
-	headerAuthorization           = "Authorization"
-	headerXmsDate                 = "x-ms-date"
-	headerContentLength           = "Content-Length"
-	headerContentEncoding         = "Content-Encoding"
-	headerContentLanguage         = "Content-Language"
-	headerContentType             = "Content-Type"
-	headerContentMD5              = "Content-MD5"
-	headerIfModifiedSince         = "If-Modified-Since"
-	headerIfMatch                 = "If-Match"
-	headerIfNoneMatch             = "If-None-Match"
-	headerIfUnmodifiedSince       = "If-Unmodified-Since"
-	headerRange                   = "Range"
-	headerDate                    = "Date"
-	headerXmsVersion              = "x-ms-version"
-	headerAcceptCharset           = "Accept-Charset"
-	headerDataServiceVersion      = "DataServiceVersion"
-	headerMaxDataServiceVersion   = "MaxDataServiceVersion"
-	headerContentTransferEncoding = "Content-Transfer-Encoding"
-
-	etagOData = "odata.etag"
-	rfc3339   = "2006-01-02T15:04:05.9999999Z"
-	timestamp = "Timestamp"
-	etag      = "ETag"
+	// BlockBlobMaxUploadBlobBytes indicates the maximum number of bytes that can be sent in a call to Upload.
+	BlockBlobMaxUploadBlobBytes = 256 * 1024 * 1024 // 256MB
+
+	// BlockBlobMaxStageBlockBytes indicates the maximum number of bytes that can be sent in a call to StageBlock.
+	BlockBlobMaxStageBlockBytes = 4000 * 1024 * 1024 // 4GB
+
+	// BlockBlobMaxBlocks indicates the maximum number of blocks allowed in a block blob.
+	BlockBlobMaxBlocks = 50000
+
+	// PageBlobPageBytes indicates the number of bytes in a page (512).
+	PageBlobPageBytes = 512
+)
 
+const (
+	headerAuthorization     = "Authorization"
+	headerXmsDate           = "x-ms-date"
+	headerContentLength     = "Content-Length"
+	headerContentEncoding   = "Content-Encoding"
+	headerContentLanguage   = "Content-Language"
+	headerContentType       = "Content-Type"
+	headerContentMD5        = "Content-MD5"
+	headerIfModifiedSince   = "If-Modified-Since"
+	headerIfMatch           = "If-Match"
+	headerIfNoneMatch       = "If-None-Match"
+	headerIfUnmodifiedSince = "If-Unmodified-Since"
+	headerRange             = "Range"
+	//headerDate                    = "Date"
```

Comment:
nit, but we can remove this commented out code

Reply:
Will be required in the code that I'm planning to add. So I wish to keep it here.